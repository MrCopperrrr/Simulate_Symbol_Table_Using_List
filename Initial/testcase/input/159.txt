INSERT a number
BEGIN
INSERT b string
BEGIN
INSERT c number
INSERT a string
RPRINT
END
END