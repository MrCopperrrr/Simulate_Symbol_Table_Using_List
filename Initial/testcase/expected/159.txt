success
success
success
success
a//2 c//2 b//1